src/transaction/log_manager.c

```diff
@@ -575,6 +583,37 @@ log_get_eof_lsa (void)
   return (&log_Gl.hdr.eof_lsa);
 }
 
+/*
+ * log_get_num_free_block -
+ *
+ * return:
+ *
+ * NOTE:
+ */
+void
+log_get_num_free_block (INT64 * num_free_block)
+{
+#if defined(SOLARIS)
+  struct statvfs buf;
+#else /* SOLARIS */
+  struct statfs buf;
+#endif /* SOLARIS */
```

Comment:
is this comment right?

Reply:
I referenced the file_io.c and log_manager.c

Reply:
I got it but please confirm it's the right comment.

Reply:
fix it.